src/main/java/uk/gov/cshr/civilservant/domain/CivilServant.java

```diff
@@ -61,6 +67,14 @@ public String getFullName() {
         return fullName;
     }
 
+    public String getLineManagerUid() {
+        return lineManagerUid;
+    }
+
+    public void setLineManagerUid(String lineManager) {
+        this.lineManagerUid =lineManager;
```

Comment:
STYLE - Spacing. You could turn on auto format on save feature from IntelliJ or format on commit to prevent small style issues.

Reply:
done

Reply:
That's not not the question,Styling requirements that have not been discussed in any way but enforced and demanded. Styling should enable just like any standard, rather than restrict and there are **many** arguments against pedantry in this approach. Saying no there's not or not or something similarly dismissive is more harmful than any simple styling issue!